test/integration/singlecluster/scheduler/fairsharing/fair_sharing_test.go

```diff
@@ -328,6 +334,73 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			expectCohortWeightedShare(cohortSecondRight.Name, 214)
 			expectCohortWeightedShare(cohortBank.Name, 0)
 		})
+		ginkgo.It("preempts workloads to enforce fair share", func() {
+			// below are Cohorts, queues, and their fair
+			// weights. 12 GPUs are provided by the root
```

Comment:
```suggestion
			// weights. 12 CPUs are provided by the root
```

Isn't this actually CPUs?

> Request(corev1.ResourceCPU, cpuRequests).Obj()

Reply:
Done.